### PR TITLE
erroneous conflicts at startup

### PR DESCRIPTION
**Pull request by exarkun**
Fixes: ticket:2965

---
This change is [Reviewable](https://reviewable.io/reviews/tahoe-lafs/tahoe-lafs/575)

### CODE REVIEW

File: integration/test_magic_folder.py [hunk 1]
```diff
@@ -395,3 +395,55 @@ def test_edmond_uploads_then_restarts(reactor, request, temp_dir, introducer_fur
...
+    yield util.await_files_exist(
+        list(
+            join(bob_magic_dir, name)
+            for name
+            in some_files
+        ),
+        await_all=True,
+    )
+    # Let it settle.
+    time.sleep(1.0)
```
Comment (meejah) [inline integration/test_magic_folder.py]: This is a use-case for some kind of status API that can answer the question "does magic-folder believe it has completed?" (Not a comment to address on this PR, this way is totally fine)

File: src/allmydata/frontends/magic_folder.py [hunk 2]
```diff
@@ -1732,7 +1743,7 @@ def start_downloading(self):
                         "Last tried at %s" % self.nice_current_time(),
                     )
                     write_traceback()
-                    yield task.deferLater(self._clock, self._poll_interval, lambda: None)
+                    yield task.deferLater(self._clock, self._scan_delay(), lambda: None)
```
Comment (meejah) [inline src/allmydata/frontends/magic_folder.py]: :+1:

**Comment by meejah** [comment]
Looks like the new test hung on CircleCI (but, I do not know why)

**Comment by exarkun** [comment]
> Looks like the new test hung on CircleCI (but, I do not know why)

Would have been too easy otherwise!

**Comment by exarkun** [comment]
Slackware and Appveyor failures are both due to https://github.com/itamarst/eliot/pull/383

**Pull request merged**